ledger-service/http-json-perf/src/main/scala/com/daml/http/perf/scenario/OracleMultiUserQueryScenario.scala

```diff
@@ -12,7 +12,7 @@ import scala.concurrent.duration._
 
 private[scenario] trait HasRandomCurrency {
   protected val rng = new scala.util.Random(123456789)
-  final val currencies = List("USD", "GBP", "EUR", "CHF", "AUD")
+  final val currencies = Range.apply(0, 1000).map(i => s"CUR_$i").toList
```

Comment:
```suggestion
  final val currencies = Range(0, 1000).map(i => s"CUR_$i")
```
The explicit `apply` is not needed and I believe the `toList` is probably not necessary:
```
scala> :t Range(0, 1000).map(i => s"CUR_$i")
IndexedSeq[String]
```

Otherwise, if you want there's ~an app for that~ `List.tabulate` that does this in the standard library:
```suggestion
  final val currencies = List.tabulate(1000)(i => s"CUR_$i")
```

Reply:
`List.tabulate` seems like the better choice thanks